Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Ref: https://jupyter-docker-stacks.readthedocs.io/en/latest/index.html
+FROM jupyter/base-notebook:python-3.8.8
```

Comment:
I decide to rely on https://github.com/jupyter/docker-stacks because I don't want to reinvent the wheel as far as running a `jupyterlab` server in docker goes.